backend/cfg/eliminate_fallthrough_blocks.ml

```diff
@@ -35,8 +35,6 @@ let is_fallthrough_block cfg_with_layout (block : C.basic_block) =
     || block.is_trap_handler
     || List.length block.body > 0
     || block.can_raise
```

Comment:
It is safe to remove the disjunct `block.can_raise`  that the comment was referring to.